components/form/DropDown.js

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+
+const DropdownMenu = ({ eventType, handleEventTypeChange, options, label, className }) => {
```

Comment:
The `Input` component has an export default on the first line, let's try to keep it consistent please

Reply:
@eddiejaoude I actually replaced the fat arrow function with the normal function as used in Input Component in the DropDown.js file to maintain consistency as you mentioned.